skeleton-es2016-aspnetcore/src/skeleton-navigation-es2016-vs/Views/Home/Index.cshtml

```diff
@@ -1,12 +0,0 @@
-﻿<div aurelia-app="main">
-    <div class="splash">
-        <div class="message">Aurelia Navigation Skeleton</div>
-        <i class="fa fa-spinner fa-spin"></i>
-    </div>
-
-    <script src="jspm_packages/system.js"></script>
-    <script src="config.js"></script>
-    <script>
-        System.import('aurelia-bootstrapper');
-    </script>
-</div>
```

Comment:
Agreed. Most people using this skeleton probably would expect the MVC to be functional to work with alongside Aurelia.  

FYI - The Url didnt have launching w/index.html in the Readme.md.  What threw me off initially. :)

Couple more things on the Readme.md
-- wrong port (9000) mentioned.
-- would mention *dotnet restore* step, if someone is using MVC bits